app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1304,7 +1304,7 @@ function (&$field) {
         ];
 
         // Send to current user
-        $errors = $model->sendEmailToUser($entity, $users, $fields, [], [], false);
+        $errors = $model->sendSampleEmailToUser($entity, $users, $fields, [], [], false);
```

Comment:
Aha, the variable names confused me. $lead is usually instance of Lead entity. It would make more sense if it was called $leadFields or $fields. But that's just a minor issue. If you'd like to update it in the sendSampleEmailToUser method before merge, I'd be grateful.